packages/grid/_modules_/grid/components/containers/GridFooterContainer.tsx

```diff
@@ -19,13 +20,26 @@ const useUtilityClasses = (ownerState: OwnerState) => {
   return composeClasses(slots, getDataGridUtilityClass, classes);
 };
 
+const GridFooterContainerRoot = styled('div', {
+  name: 'MuiDataGrid',
+  slot: 'FooterContainer',
+  overridesResolver: (props, styles) => styles.footerContainer,
+})(() => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  minHeight: 52, // Match TablePagination min height
+}));
+
 export const GridFooterContainer = React.forwardRef<HTMLDivElement, GridFooterContainerProps>(
   function GridFooterContainer(props: GridFooterContainerProps, ref) {
     const { className, ...other } = props;
     const rootProps = useGridRootProps();
     const ownerState = { classes: rootProps.classes };
     const classes = useUtilityClasses(ownerState);
 
-    return <div ref={ref} className={clsx(classes.root, className)} {...other} />;
```

Comment:
Something is off with the `classes`. `gridClasses` has `footerContainer` and yet, here, we create the class name a second time.

I think that we should either:

1. have `<DataGrid classes={{ footerContainer: 'foo' }} />` to actually work and use `gridClasses.footerContainer` here
2. or change the name of the styled component: `name: 'MuiDataGrid',` -> `name: 'GridFooterContainer',`
  `slot: 'FooterContainer',` -> `slot: 'Root',`

Reply:
I would stick to the first option since that is the convention we are currently following. IMO it will look off to have `GridFooterContainer`  because that way we would need to go through the other components and update them as well.

Reply:
@oliviertassinari  I didn't understand the problem and solution proposed in the first option.

- The value of `gridClasses.footerContainer` is `MuiDataGrid-footerContainer` which breaks `composeClasses`.

- `<DataGrid classes={{ footerContainer: 'foo' }} />` works correctly: 

    <img width="600" src="https://user-images.githubusercontent.com/42154031/140550927-48e13384-a5d7-4d2a-91ed-075f02067e34.png" alt="image" style="max-width: 100%;">

Reply:
Ok, it seems that we are good.